docs/content/preview/architecture/docdb-replication/async-replication.md

```diff
@@ -275,12 +275,15 @@ Refer to [Inconsistencies affecting transactions](#inconsistencies-affecting-tra
 - In Semi-automatic and Manual modes, schema changes are not automatically replicated. They must be manually applied to both source and target universes. Refer to [DDLs in semi-automatic mode](../../../deploy/multi-dc/async-replication/async-transactional-setup-semi-automatic/#making-ddl-changes) and [DDLs in manual mode](../../../deploy/multi-dc/async-replication/async-transactional-tables) for more information.
 
 ### Transactional Automatic mode limitations
+- All connections to the source universe must be reset after setting up the replication. [#25853](https://github.com/yugabyte/yugabyte-db/issues/25853)
```

Comment:
nit: could group/order these points a bit better
- connections
- global objs
- unsupported ddls
- extensions

Reply:
Made it
- temp limitations
- globals
- unsupported ddls
- extensions